js/utils/constants.js

```diff
@@ -2287,12 +2287,12 @@
          */
         VIDEO: 'video',
         /**
-         * @description <code>OO.CONSTANTS.PLAYER_TYPE.AUDIO ('audio')</code>:
+         * @description <code>OO.CONSTANTS.PLAYER_TYPE.AUDIO ('audio_only')</code>:
          *   The audio-only player type.
          * @constant OO.CONSTANTS.PLAYER_TYPE.AUDIO
          * @type {string}
          */
-        AUDIO: 'audio',
+        AUDIO: 'audio_only',
```

Comment:
We should check with Steve, but I believe the intent was for the player to have the page level param "playerType": "audio" rather than "audio_only"

Reply:
@aeng7 I changed it since it looks like JP was sending this constant as the query param value to SAS, and I think they are expecting `audio_only` there

Reply:
I'll ping Steve though

Reply:
Sounds good. Just want to make sure we're on the same page. See https://jira.corp.ooyala.com:8443/browse/PLAYER-4385

Reply:
@aeng7  I reverted it after talking to Nagesh, he is already using 'audio' as the value so I don't want to break anything of his. We might have to ask SAS to change what they expect in the query param, but I think we can do a workaround for now if we need to